### ISSUE TITLE
Destroy event handlers getting called twice

### ISSUE THREAD

**Issue by m3g4p0p** [replit/kaboom#31]
Hi there,

I just noticed that since version 0.3.0 destroy event handlers are getting called twice. To reproduce with 0.4.0:

```javascript
const k = kaboom()

k.scene('main', () => {
  const foo = k.add(['foo'])
  
  k.on('destroy', 'foo', console.log)
  k.destroy(foo)
})

k.start('main')
```

Cheers, having a lot of fun with your library! :)

**Comment by slmjkdbtl** [comment]
Indeed it is, nice catch! Fixed in c80bf46.

And thanks! Glad you enjoyed

**Comment by m3g4p0p** [comment]
Awesome, thank you too! :)
